### ISSUE TITLE
Typo in the docstring of HostnameEndpoint._endpoints

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6729]
|[<img alt="ashfall's avatar" src="https://avatars.githubusercontent.com/u/1138873?s=50" width="50" height="50">](https://github.com/ashfall)| @ashfall reported|
|-|-|
|Trac ID|trac#6729|
|Type|defect|
|Created|2013-09-07 13:10:49Z|
|Branch|https://github.com/twisted/twisted/tree/fix-typo-hostname-endpoint-6729|

"famliy" should be "family" [in the docstring](http://twistedmatrix.com/trac/browser/trunk/twisted/internet/endpoints.py#L692)

<details><summary>Searchable metadata</summary>

```
trac-id__6729 6729
type__defect defect
reporter__ashfall ashfall
priority__normal normal
milestone__ 
branch__branches_fix_typo_hostname_endpoint_6729 branches/fix-typo-hostname-endpoint-6729
branch_author__ashfall ashfall
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1378559449000000 1378559449000000
changetime__1378732595000000 1378732595000000
version__None None
owner__ashfall ashfall

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="ashfall's avatar" src="https://avatars.githubusercontent.com/u/1138873?s=50" width="50" height="50">](https://github.com/ashfall)<a name="note_1"></a>|@ashfall commented|
|-|-|

(In [39894]) Branching to 'fix-typo-hostname-endpoint-6729'

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)<a name="note_3"></a>|@hynek set owner to @ashfall|
|-|-|

Ready for merge. :)

**Comment by twisted-trac** [comment]
|[<img alt="ashfall's avatar" src="https://avatars.githubusercontent.com/u/1138873?s=50" width="50" height="50">](https://github.com/ashfall)<a name="note_4"></a>|@ashfall set status to `closed`|
|-|-|

(In [39896]) Merge fix-typo-hostname-endpoint-6729: Fix a typo in the doctring of HostnameEndpoint._endpoints

Author: ashfall
Reviewer: hynek
Fixes: [#6729](https://github.com/twisted/twisted/issues/6729)
